cmd/up/logs.go

```diff
@@ -59,9 +56,9 @@ func (up *upContext) showDetachedLogs(ctx context.Context) error {
 		}
 	}()
 	select {
-	case <-stop:
-		oktetoLog.Infof("CTRL+C received, starting shutdown sequence")
-		return oktetoErrors.ErrIntSig
+	case <-ctx.Done():
```

Comment:
we should only handle `os.Interrupt` at one place. `os.Interrupt` is handled at the beginning of `okteto up`. There, the context is cancelled and we notified on `ctx.Done()`.